### ISSUE TITLE
Fix typo in setup.py

### ISSUE THREAD

**Issue by bikegeek** [dtcenter/METcalcpy#43]
## Describe the Problem ##
The setup.py has mis-spelled the metcalcpy package name, it is 'metcalpy', missing it's second 'c'

### Expected Behavior ###
in the setup.py module, fix the typo 

### Environment ###
Describe your runtime environment:
*1. Machine: Mac Laptop
*2. OS:  MacOS
*3. Software version number(s)*:
name: blenny_363
channels:
  - plotly
  - anaconda
  - conda-forge
  - defaults
dependencies:
  - alabaster=0.7.12=py_0
  - appdirs=1.4.3=py_1
  - appnope=0.1.0=py36h9f0ad1d_1001
  - asn1crypto=1.3.0=py36h9f0ad1d_1
  - astroid=2.3.3=py36_1
  - atomicwrites=1.4.0=pyh9f0ad1d_0
  - attrs=19.3.0=py_0
  - babel=2.8.0=py_0
  - backcall=0.2.0=pyh9f0ad1d_0
  - basemap=1.2.1=py36hee42265_1
  - bleach=3.1.5=pyh9f0ad1d_0
  - bokeh=2.1.1=py36h9f0ad1d_0
  - brotlipy=0.7.0=py36h37b9a7d_1000
  - bzip2=1.0.8=h0b31af3_2
  - ca-certificates=2020.6.20=hecda079_0
  - cairo=1.16.0=ha8983da_1005
  - cartopy=0.17.0=py36hd103db2_1006
  - certifi=2020.6.20=py36h2432673_2
  - cffi=1.14.0=py36h356ff06_0
  - cftime=1.2.1=py36ha042823_0
  - chardet=3.0.4=py36h9f0ad1d_1006
  - click=7.1.2=pyh9f0ad1d_0
  - cloudpickle=1.5.0=py_0
  - cryptography=2.5=py36hdbc3d79_1
  - curl=7.64.0=heae2a1f_0
  - cycler=0.10.0=py_2
  - cytoolz=0.10.1=py36h0b31af3_0
  - dask=2.5.2=py_0
  - dask-core=2.5.2=py_0
  - decorator=4.4.2=py_0
  - defusedxml=0.6.0=py_0
  - distributed=2.5.2=py_0
  - docutils=0.16=py36h9f0ad1d_1
  - eccodes=2.12.3=hac8aea4_1
  - entrypoints=0.3=py36h9f0ad1d_1001
  - ffmpeg=4.1.3=h5c2b479_0
  - fontconfig=2.13.1=h79c0d67_1002
  - freetype=2.10.2=h8da9a1a_0
  - fsspec=0.7.4=py_0
  - geos=3.7.2=h6de7cb9_2
  - gettext=0.19.8.1=h46ab8bc_1002
  - giflib=5.2.1=h0b31af3_2
  - glib=2.65.0=h577aef8_0
  - gmp=6.2.0=hb1e8313_2
  - gnutls=3.6.13=hc269f14_0
  - graphite2=1.3.13=h12caacf_1001
  - graphviz=2.38.0=hc6cc99f_1011
  - harfbuzz=2.7.2=h8810732_0
  - hdf4=4.2.13=hf3c6af0_1002
  - hdf5=1.10.4=nompi_h0cbb7df_1106
  - heapdict=1.0.1=py_0
  - icu=67.1=h4a8c4bd_0
  - idna=2.10=pyh9f0ad1d_0
  - imageio=2.9.0=py_0
  - imagesize=1.2.0=py_0
  - importlib-metadata=1.7.0=py36h9f0ad1d_0
  - importlib_metadata=1.7.0=0
  - imutils=0.5.3=py36h9f0ad1d_2
  - invoke=1.4.1=py_0
  - ipykernel=5.3.4=py36h95af2a2_0
  - ipython=7.16.1=py36h95af2a2_0
  - ipython_genutils=0.2.0=py_1
  - isort=4.3.21=py36h9f0ad1d_1
  - jasper=1.900.1=h636a363_1006
  - jedi=0.16.0=py36h9f0ad1d_1
  - jinja2=2.11.2=pyh9f0ad1d_0
  - jpeg=9d=h0b31af3_0
  - json5=0.9.4=pyh9f0ad1d_0
  - jsonschema=3.2.0=py36h9f0ad1d_1
  - jupyter_client=6.1.6=py_0
  - jupyter_core=4.6.3=py36h9f0ad1d_1
  - jupyterlab=1.2.6=py_0
  - jupyterlab_server=1.2.0=py_0
  - kiwisolver=1.2.0=py36h863e41a_0
  - krb5=1.16.3=h24a3359_1000
  - lame=3.100=h1de35cc_1001
  - lazy-object-proxy=1.5.1=py36h9de38fb_0
  - lcms2=2.11=h174193d_0
  - libaec=1.0.4=h4a8c4bd_1
  - libblas=3.8.0=14_openblas
  - libcblas=3.8.0=14_openblas
  - libcurl=7.64.0=h76de61e_0
  - libcxx=10.0.1=h5f48129_0
  - libedit=3.1.20191231=hed1e85f_1
  - libffi=3.2.1=h4a8c4bd_1007
  - libgfortran=3.0.1=0
  - libiconv=1.15=h0b31af3_1006
  - liblapack=3.8.0=14_openblas
  - liblapacke=3.8.0=14_openblas
  - libnetcdf=4.6.2=h6b88ef6_1001
  - libopenblas=0.3.7=hd44dcd8_1
  - libpng=1.6.37=hbbe82c9_1
  - libsodium=1.0.17=h01d97ff_0
  - libssh2=1.8.0=hf30b1f0_1003
  - libtiff=4.1.0=ha78913b_3
  - libwebp=1.0.2=hd3bf737_5
  - libxml2=2.9.10=hc06c4ae_1
  - libxslt=1.1.33=h29b7fa6_1
  - locket=0.2.0=py_2
  - lxml=4.4.1=py36h08abf6f_0
  - lz4-c=1.9.2=h4a8c4bd_1
  - markupsafe=1.1.1=py36h37b9a7d_1
  - matplotlib=3.3.1=1
  - matplotlib-base=3.3.1=py36h534ab7b_1
  - mccabe=0.6.1=py_1
  - metpy=0.11.0=py36_1
  - mistune=0.8.4=py36h37b9a7d_1001
  - more-itertools=8.4.0=py_0
  - msgpack-python=0.6.2=py36ha1b3eb9_0
  - nbconvert=5.6.1=py36h9f0ad1d_1
  - nbformat=5.0.7=py_0
  - ncurses=6.2=hb1e8313_1
  - netcdf4=1.5.1.2=py36h58b02fa_0
  - nettle=3.4.1=h3efe00b_1002
  - networkx=2.5=py_0
  - nodejs=6.13.1=0
  - nose=1.3.7=py36h9f0ad1d_1004
  - notebook=6.0.3=py36h9f0ad1d_1
  - numpy=1.19.0=py36h4a66613_0
  - olefile=0.46=py_0
  - opencv=4.1.0=py36h3012f30_4
  - openh264=1.8.0=hd9629dc_1000
  - openssl=1.0.2u=h0b31af3_0
  - owslib=0.19.2=py_0
  - packaging=20.4=pyh9f0ad1d_0
  - pandas=1.0.5=py36hcc1bba6_0
  - pandoc=2.10=h1de35cc_0
  - pandocfilters=1.4.2=py_1
  - partd=1.1.0=py_0
  - pathspec=0.8.0=pyh9f0ad1d_0
  - patsy=0.5.1=py_0
  - pcre=8.44=h4a8c4bd_0
  - pexpect=4.8.0=py36h9f0ad1d_1
  - pickleshare=0.7.5=py36h9f0ad1d_1001
  - pillow=7.2.0=py36h2ae5dfa_1
  - pint=0.14=py_0
  - pip=20.1.1=py_1
  - pixman=0.38.0=h01d97ff_1003
  - plotly=4.9.0=py_0
  - pluggy=0.13.1=py36h9f0ad1d_2
  - pooch=1.1.1=py_0
  - proj4=5.2.0=h6de7cb9_1006
  - prometheus_client=0.8.0=pyh9f0ad1d_0
  - prompt-toolkit=3.0.5=py_1
  - psutil=5.7.2=py36h9de38fb_0
  - ptyprocess=0.6.0=py_1001
  - py=1.9.0=pyh9f0ad1d_0
  - pycparser=2.20=pyh9f0ad1d_2
  - pyepsg=0.4.0=py_0
  - pygments=2.6.1=py_0
  - pygrib=2.0.4=py36h4e57d6e_0
  - pykdtree=1.3.1=py36h255dfe6_1003
  - pylint=2.4.2=py36_0
  - pymysql=0.9.3=py_0
  - pyopenssl=19.0.0=py36_0
  - pyparsing=2.4.7=pyh9f0ad1d_0
  - pyproj=1.9.6=py36h0b31af3_1002
  - pyrsistent=0.16.0=py36h37b9a7d_0
  - pyshp=2.1.0=py_0
  - pysocks=1.7.1=py36h9f0ad1d_1
  - pytest=5.2.1=py36_0
  - pytest-mpl=0.10=py_0
  - pytest-regtest=1.4.2=py_0
  - python=3.6.3=4
  - python-dateutil=2.8.1=py_0
  - python-kaleido=0.0.1=py36_0
  - python_abi=3.6=1_cp36m
  - pytz=2020.1=pyh9f0ad1d_0
  - pywavelets=1.1.1=py36h1de35cc_0
  - pyyaml=5.3.1=py36h37b9a7d_0
  - pyzmq=19.0.1=py36h820b253_0
  - readline=7.0=hcfe32e1_1001
  - requests=2.24.0=pyh9f0ad1d_0
  - retrying=1.3.3=py_2
  - scikit-image=0.16.2=py36h6c726b0_0
  - scipy=1.3.1=py36hab3da7d_2
  - seaborn=0.10.0=py_1
  - setuptools=49.2.0=py36h9f0ad1d_0
  - shapely=1.6.4=py36h5c88e11_1006
  - six=1.15.0=pyh9f0ad1d_0
  - snowballstemmer=2.0.0=py_0
  - sortedcontainers=2.2.2=pyh9f0ad1d_0
  - sphinx=2.2.0=py_0
  - sphinx-gallery=0.4.0=py36_0
  - sphinx_rtd_theme=0.4.3=py_0
  - sphinxcontrib-applehelp=1.0.2=py_0
  - sphinxcontrib-devhelp=1.0.2=py_0
  - sphinxcontrib-htmlhelp=1.0.3=py_0
  - sphinxcontrib-jsmath=1.0.1=py_0
  - sphinxcontrib-qthelp=1.0.3=py_0
  - sphinxcontrib-serializinghtml=1.1.4=py_0
  - sqlite=3.20.1=0
  - statsmodels=0.11.1=py36h37b9a7d_2
  - tblib=1.6.0=py_0
  - terminado=0.8.3=py36h9f0ad1d_1
  - testpath=0.4.4=py_0
  - tk=8.6.10=hbbe82c9_0
  - toolz=0.10.0=py_0
  - tornado=6.0.4=py36h37b9a7d_1
  - traitlets=4.3.3=py36h9f0ad1d_1
  - typed-ast=1.4.1=py36h0b31af3_0
  - typing=3.7.4.3=py36h9f0ad1d_1
  - typing_extensions=3.7.4.2=py_0
  - urllib3=1.25.10=py_0
  - wcwidth=0.2.5=pyh9f0ad1d_0
  - webencodings=0.5.1=py_1
  - wheel=0.34.2=py_1
  - wrapt=1.12.1=py36h37b9a7d_1
  - x264=1!152.20180806=h1de35cc_0
  - xarray=0.16.0=py_0
  - xlsxwriter=1.2.8=py_0
  - xz=5.2.5=h0b31af3_1
  - yaml=0.2.5=h0b31af3_0
  - yamllint=1.15.0=py_1
  - zeromq=4.3.2=h6de7cb9_2
  - zict=2.0.0=py_0
  - zipp=3.1.0=py_0
  - zlib=1.2.11=h0b31af3_1006
  - zstd=1.4.5=h0384e3a_1
  - pip:
    - altgraph==0.17
    - asttokens==2.0.3
    - bootstrapped==0.0.2
    - chart-studio==1.0.0
    - joblib==0.14.1
    - macholib==1.14
    - mypy==0.761
    - mypy-extensions==0.4.3
    - parso==0.6.1
    - pyinstaller==3.6
    - pyserial==3.4
    - scikit-learn==0.22.2
    - send2trash==1.5.0
    - sklearn==0.0
    - typing-extensions==3.7.4.1

### To Reproduce ###
Describe the steps to reproduce the behavior:
*1. Working in your conda environment, re-created using the packages exported from current conda environment (see above)
*2. Clone the develop branch of the METcalcpy repository
*3. From the top directory of METcalcpy (where the setup.py module is located), run:
      pip install -e . 
     to install the metcalcpy package into the current conda environment
*4. from the command line, enter conda list|grep met
*5. Scroll down and find 'metcalpy' :
importlib-metadata        1.7.0            py36h9f0ad1d_0    conda-forge
importlib_metadata        1.7.0                         0    conda-forge
metcalpy                  0.1                       dev_0    <develop>
metpy                     0.11.0                   py36_1    conda-forge
prometheus_client         0.8.0              pyh9f0ad1d_0    conda-forge

Time Estimate:   ~1 hour

### To Fix ###
*1. Open setup.py
*2. replace name="metcalpy" with name="metcalcpy"
*3. merge into 'develop' branch

### To test ###
*1. Load new METcalcpy development  into your current conda environment:
     *a: from directory where setup.py resides, enter the following:
            pip install -e .
*2. Run the PYTHON REPL and import as follows:
     *a: from metcalcpy.util import utils
     *b: utils.round_half_up(1.5)
           should return the value 2 

### Relevant Deadlines ###
Beta2

### Assignee ###
- [x] Select **engineer(s)**Minna Win
- [x] Select  **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [x] Select **milestone** to relevant bugfix version

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Bugfix Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.

Branch name: `bugfix_<Issue Number>_main_<Version>_<Description>`
- [x] Fix the bug and test your changes.
- [x] Commit  changes to GitHub develop branch.

- [x] Close this issue.

**Comment by bikegeek** [comment]
change made in develop branch
